content/1.7/back-office/shipping/carriers/Add-edit-carrier.md

```diff
@@ -0,0 +1,202 @@
+If one of the ranges overlaps another, the following error message is displayed: "Ranges are overlapping" and the fields for all zones are disabled.
+    - **All** This field is disabled if the checkbox isn't checked. When the checkbox is checked, the field is enabled and all the zones below become checked"
+ 
+All the zones existing in International > Locations > Zones are displayed below. The disabled zones are marked as inactive. 
+Each zone has a checkbox allowing to enable its field.
+Only numbers are allowed in zones fields, inputting any other symbol will result in the field being marked in red. 
+
+### Call to action 
+
+There is a button below the table "Add new range". When the button is pressed, a new row near the "ranges" appear, the table is the same as 6th field, only the "All" row is not duplicated. With the duplicate table a button "Delete" also appears below it.
+
+  - **Previous.** In the first step, this button is disabled. After clicking on it, the previous step is displayed.
+  - **Next.** If there is an error, this button is disabled. After clicking on it, the next step is displayed.
+  - Finish. Non pressable until all the pages mandatory fields are input correctly. When all mandatory fields are input correctly, takes to the [carriers listing]({{<ref "carriers-listing.md">}} "Carriers listing") page.
+
+# Size, weight, and group access (4th step)
+
+### Call to action
+ 
+A call to action button at the top right is displayed:
+
+ - Cancel:
+Pressing the button redirects to the [carriers listing]({{<ref "carriers-listing.md">}} "Carriers listing") page.
+
+## Columns
+
+Input fields only accept numbers, when trying to add any other symbols, and trying to proceed to next page, an alert is prompted showing:
+"TECHNICAL ERROR:
+Details:
+Error thrown: [object Object]
+Text status: error"
+And an option to press "Ok".
+
+ - Maximum package width (cm) an input field, has a tooltip: "Mazimum width managed by this carrier. Set the value to "0", or leave this field blank to ignore. The value must be an integer."
+ - Maximum package height (cm) an input field, has a tooltip: "Maximum height managed by this carrier. Set the value to "0", or leave this field blank to ignore. The value must be an integer."
+ - Maximum package depth (cm) an input field, has a tooltip: "Maximum height managed by this carrier. Set the value to "0", or leave this field blank to ignore. The value must be an integer."
+ - Maximum package weight (Kg) an input field, has a tooltip: "Maximum height managed by this carrier. Set the value to "0", or leave this field blank to ignore."
+ - Group access, displays the ID and group name in a table, all of them have a checkmark on the left, by default Vistor, guest and customer has checkmarks checked.
+A tooltip is displayed: "Mark the groups that are allowed access to this carrier."
+     - Group name, if this field id checkmarked, all the fields get checkmarked or checkmark gets removed.
+     - Visitor
```

Comment:
```suggestion
```

Reply:
We should avoid listing the group because it depends on the configuration: https://github.com/PrestaShop/prestashop-specs/pull/272/files#r769490767
> The ID and the name of all existing customers' groups are displayed.